src/Utilities/UnitTests/ToolTask_Tests.cs

```diff
@@ -27,7 +27,11 @@ public MyTool()
                 : base()
             {
                 _fullToolName = Path.Combine(
+#if FEATURE_SPECIAL_FOLDERS
                     Environment.GetFolderPath(Environment.SpecialFolder.System),
```

Comment:
I see this pattern a lot. Could we use an abstraction in the FileUtilities class and #if there and always use that? We can do that later, just seems like it might be easier.